docs/BuildController.md

```diff
@@ -0,0 +1,36 @@
+# BuildController
+
+## This makes a bunch of imagestreams, what are they for? 
+
+|imagestream| intended purpose|
+|---|---|
+|`coreos`                            | The global default base coreos image (rhel-coreos) |
+|`layered-base`                    |   The base image stream our buildconfigs draw from (defaults to coreos:latest) |
+|`layered-external-base`           |External base image stream. If :latest tag is populated, layered-base uses this instead of coreos:latest|  
+|`layered-mco-content`        | This is the image built by our mco dockerfile where ignition-apply runs |
```

Comment:
One of the other conversations we also had around this was whether or not the "custom pools" should derive from the worker pool image or if they should build their own. So right now on the layering branch:
-  each pool in "layred/image mode" (custom or not) gets its own set of imagestreams and buildconfigs. 

This is okay because the render controller is merging the config for us --  it's handling "config derivation" on the machineconfig front for custom pools, but the _image_ content is still being built 'from base'. 

I'm just writing this here so we remember to think through whether "all custom pools ~inherit~ derive from worker" is a meme that we want to carry forward into our new derived image world, or if it is something we only did out of necessity because the pool needed the config in worker (base, templates, etc) to live ?